src/aleph_client/asynchronous.py

```diff
@@ -129,56 +130,129 @@ async def storage_push_file(
         return (await resp.json()).get("hash")
 
 
+def _log_publication_status(publication_status: Mapping[str, Any]):
```

Comment:
Isn't the fact that a method prefixed with `_log_` raises an exception confusing ?

Reply:
It only raises a `ValueError` if the API response has the wrong format. We need better error handling in this repository but until then I'd say that this is fair.